src/ex1/styles/buttons.css

```diff
@@ -0,0 +1,50 @@
+@import "button-hover.css";
+
+button {
+  cursor: pointer;
+}
+
+#add-todo-button {
+  background-color: var(--dark-green);
+  border-radius: 4px;
+  border: 0px;
+  color: antiquewhite;
+  height: 45px;
+  width: 10%;
+  font-size: 32px;
+}
+
+.remove-todo-button {
+  background-color: brown;
+  border: 0px;
+  border-radius: 4px;
+  color: antiquewhite;
+  height: 40px;
+  width: 10%;
```

Comment:
In most cases, you probably don't want the button size to change every time the UI it's rendered in changes, so I would suggest using a fixed width to improve maintainability

Reply:
done